scvi_in_R.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>feature plot in a new cell, all the umap plots with expression should be the lognormalized data, so you will have to run lognormalization in seurat, which is why you would use the counts option in getassaydata</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>